test/unit-tests/expression/parse.test.js

```diff
@@ -1465,6 +1465,10 @@ describe('parse', function () {
       assert.strictEqual(parseAndEval('true & false'), 0)
       assert.strictEqual(parseAndEval('false & true'), 0)
       assert.strictEqual(parseAndEval('false & false'), 0)
+
+      assert.strictEqual(parseAndEval('0 & undefined'), 0)
```

Comment:
Using undefined here is a nifty way to verify that the second condition is not executed 😄, nice! Indeed we need to verify that `undefined` is not supported too and you did that with the `assert.throws(...)`.